dv/parameters/KDV01_CubeOrangePlusDefault.param

```diff
@@ -832,13 +897,13 @@ SERIAL0_PROTOCOL,2.0
 SERIAL1_BAUD,57.0
```

Comment:
I don't know what SERIAL_1_BAUD is set to on the drone in Paris with a payload. But 57kBaud is quite slow.

Reply:
@SietseAT , is there any reason we go with 57?
I propose to change it to 921 instead.